tests/test_deployments_button_QCI93.py

```diff
@@ -0,0 +1,5 @@
+#QCI-93
```

Comment:
Could we add some text to explain what it is we are testing.   I realize we have a reference to a Polarion ID but someday we won't use Polarion ande that reference may or may not be meaningful.

I would also provide a full link to the Polarian test case.